src/lib/openapi/standard-responses.ts

```diff
@@ -0,0 +1,23 @@
+import { OpenAPIV3 } from 'openapi-types';
```

Comment:
Maybe rename this file to something like `operations.ts` and move some stuff from `index.ts`:

* ApiOperation
* createRequestSchema
* createResponseSchema

Reply:
Hmm, we could, but maybe it's better to do the opposite and move this into index instead? Or maybe call the file "utils" or "helpers" or something? I feel like "operations" is a bit too vague 🤷🏼

Reply:
True, pretty vague. Moving it into `index.ts` could work. Would be good to split that file up soon though. Maybe we should have one file per helper, and keep the schemas list in `index.ts`. Then we'd have specific file names at least.

Reply:
Yeah, I see what you mean. Renegade suggestion: move the helpers into separate file and re-export them from index? Maybe? Or do that with a utils subdirectory?

Reply:
Good idea! Maybe `openapi/util` to match the other `lib/util` dir name? 

I don't think we need to re-export from an index though, we usually just import stuff directly.

Reply:
Got it! Yeah, I can do that 🙋🏼

Reply:
Actually, on second thought: I think we should do that in a separate PR. I'm happy to put the new stuff in there, but I don't really want to touch another bajillion files as part of this PR. What do you think?

Reply:
Agreed, good idea. :)